test_vitals.py

```
@@ -0,0 +1,44 @@
+import json
+
+if len(sys.argv) < 2:
+    print("Usage: python test_vitals.py <password>")
+    print("Example: python test_vitals.py admin_secret_123")
+    sys.exit(1)
+
+password = sys.argv[1]
+url = "http://localhost:15090/api/admin/vitals"
+
+try:
+    response = requests.get(url, headers={"Authorization": f"Bearer {password}"})
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)